doc/source/whatsnew/v0.24.2.rst

```diff
@@ -66,7 +66,7 @@ Bug Fixes
 
 **Categorical**
 
--
+- Bug where a copy of a categorical could have the wrong dimensions (:issue:`24971`)
```

Comment:
can you reword to make this more clear. mentione this is on a Series of categorical dtype. The dimenension issue is an internal one. You want to emphasize the user visible effects.

Reply:
this is visible when you call ``.replace()``